misk/src/main/kotlin/misk/MiskCaller.kt

```diff
@@ -9,7 +9,11 @@ data class MiskCaller(
   val user: String? = null,
 
   /** Set of roles to which the human user belongs, typically provided by the SSO infrastructure */
-  val roles: Set<String> = setOf()
+  // TODO(rhall): Deprecate this https://github.com/cashapp/misk/issues/1078
```

Comment:
it's kind of lame that you can add `@Deprecated` with some kind of INFO level that won't break builds that have `-WError` set

Reply:
I think `-WError` sucks